CRM/Upgrade/Incremental/sql/5.79.alpha1.mysql.tpl

```diff
@@ -1 +1,3 @@
 {* file to handle db changes in 5.79.alpha1 during upgrade *}
+UPDATE `civicrm_financial_type` SET `label` = `name` WHERE `label` = '';
+UPDATE `civicrm_financial_account` SET `label` = `name` WHERE `label` = '';
```

Comment:
Because the field is localizable the sql statement will need to be localized, like:

https://github.com/civicrm/civicrm-core/blob/002fb4a236ebec13a0b270af8f05364cd4bc37c4/CRM/Upgrade/Incremental/sql/5.61.alpha1.mysql.tpl#L7-L8